src/rules.ruleset

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0"?>
+<RuleSet Name="All Rules Enabled with default severity" Description="All Rules are enabled with default severity. Rules with IsEnabledByDefault = false are force enabled with default severity." ToolsVersion="15.0">
+   <Rules AnalyzerId="Microsoft.CodeAnalysis.FxCopAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.FxCopAnalyzers">
```

Comment:
How was this ruleset chosen?

Reply:
Do we want to turn the ones that currently aren't being hit to Errors?

Reply:
We took the global ruleset that Microsoft suggests.

I think they are almost all being hit :) We don't have a list of cross referencing failures vs. what exists.

Reply:
That's fair, generating that sounds like more trouble than it would be worth